aspnetcore/whats-new/2019-09.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "ASP.NET Core docs: What's new (September 2019)"
+description: "What's new in ASP.NET Core docs in September 2019."
+ms.date: 01/17/2020
+---
+
+# ASP.NET Core docs: What's new (September 2019)
```

Comment:
```suggestion
# ASP.NET Core docs: What's new September 2019
```

Reply:
Per MT style guide, avoid () when you can.

Reply:
> # ASP.NET Core docs: What's new for September 2019

Adding `for` sounds better. No?

Reply:
Updating the code right now to incorporate this suggestion.